packages/css/native/shared/Css_AtomicTypes.ml

```diff
@@ -942,13 +951,53 @@ module ColorMixMethod = struct
     ]
 end
 
+module Percentage_CalcMinMax = struct
```

Comment:
I don't understand this type, what does map into?

Reply:
> calc, min, max functions

According to the colorMix, spec, the above can't be applied to the percentage values.

Reply:
No, there is not. It's just `Percentage` (which is calculable)

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl#formal_syntax

Reply:
I don't understand then, why do we need another module for `percentage values that can be applied to calc,min, max functions` when Percentage already works with calc functions?

Should this definition change if minmax support Percentage? https://github.com/davesnx/styled-ppx/pull/411/files#diff-cfe53d320cde68df2b3e360ef1ec42c31e2f8077cf8b512eedd467b9fbb70d7dR97-R98

Reply:
Right, but all modules in this file tries to match with a concept in CSS. `Percentage_CalcMinMax` it isn't.

If there's some restriction in color's alpha, let's keep this type inside the Color module